src/button/button-group.scss

```diff
@@ -106,3 +106,8 @@
     }
   }
 }
+
+@mixin iui-button-group-overflow-x {
+  max-width: revert;
```

Comment:
Remind again, why max-width is always needed?

Reply:
Hmm I think it could be removed now actually